site/en/docs/crux/historical-api/index.md

```diff
@@ -0,0 +1,591 @@
+        <p>The form factor is the device class that all users used to access the site for this record.</p><p>If the form factor is unspecified, then aggregated data over all form factors will be returned.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code translate="no" dir="ltr">effectiveConnectionType</code></td>
+      <td>
+        <p><strong><code class="apitype" translate="no" dir="ltr">string</code></strong></p>
+        <p>The effective connection type is the general connection class that all users experienced for this record. This field uses the values ["offline", "slow-2G", "2G", "3G", "4G"] as specified in: <a href="https://wicg.github.io/netinfo/#effective-connection-types">https://wicg.github.io/netinfo/#effective-connection-types</a></p><p>If the effective connection type is unspecified, then aggregated data over all effective connection types will be returned.</p>
+      </td>
+    </tr>
+    <tr>
+      <td colspan="2">Union field <code translate="no" dir="ltr">url_<wbr>pattern</code>. The url pattern is the url that the record applies to. <code translate="no" dir="ltr">url_<wbr>pattern</code> can be only one of the following:</td>
+    </tr>
+    <tr>
+      <td><code translate="no" dir="ltr">origin</code></td>
+      <td>
+        <p><strong><code class="apitype" translate="no" dir="ltr">string</code></strong></p>
+        <p>Origin specifies the origin that this record is for.</p><p>Note: When specifying an origin, data for loads under this origin over all pages are aggregated into origin level user experience data.</p>
+      </td>
+    </tr>
+    <tr>
+      <td><code translate="no" dir="ltr">url</code></td>
+      <td>
+        <p><strong><code class="apitype" translate="no" dir="ltr">string</code></strong></p>
+        <p>Url specifies a specific url that this record is for.</p><p>Note: When specifying a "url" only data for that specific url will be aggregated.</p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+#### Metrics {: #api-response-metrics }
+
+A `metric` is a set of user experience data for a single web performance metric, such as first contentful paint. It contains a summary histogram of real world Chrome usage as a series of `bins`.
+
+```json
+{
+  "histogramTimeseries": [
+    {
+      object (Bin)
+    }
+  ],
+  "percentilesTimeseries": {
+    object (Percentiles)
+  }
+}
+```
+
+<table class="width-full with-heading-tint">
+  <colgroup>
+    <col width="25%">
+    <col>
+  </colgroup>
+  <thead>
+    <tr>
+      <th colspan="2">Fields</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code translate="no" dir="ltr">histogramTimeseries[]</code></td>
+      <td>
+        <p><strong><code class="apitype" translate="no" dir="ltr">object (<a href="#api-response-bin">Bin</a>)</code></strong></p>
+        <p>The timeseries histogram of user experiences for a metric. The timeseries histogram will have at least one bin and the densities of all bins will add up to ~1.</p>
```

Comment:
Maybe add that when a histogram bin density value is missing from a time series, that's indicated as NaN. In this case, all three corresponding densities will be NaN.